install/installer/pkg/components/usage/configmap_test.go

```diff
@@ -20,5 +20,19 @@ func TestConfigMap_ContainsSchedule(t *testing.T) {
 	cfgmap, ok := objs[0].(*corev1.ConfigMap)
 	require.True(t, ok)
 
-	require.JSONEq(t, cfgmap.Data[configJSONFilename], `{"controllerSchedule": "2m", "stripeCredentialsFile": "stripe-secret/apikeys"}`)
+	require.JSONEq(t,
+		`{
+       "controllerSchedule": "2m",
+       "stripeCredentialsFile": "stripe-secret/apikeys",
+       "server": {
+         "services": {
+           "grpc": {
+             "address": ":9001",
+             "tls": null
```

Comment:
I think you should be able to omit this

Reply:
Makes sense, wanna make that change?

Reply:
I'm going to do it in another PR because doing so will tag the workspace team into this one as reviewers which doesn't seem worth it.

Reply:
here: https://github.com/gitpod-io/gitpod/pull/11240